WordPress/gutenberg
# Implement roving tabindex on the Image block

This PR is part of #18619, whose main goal is to implement [roving tabindex](https://www.w3.org/TR/wai-aria-practices/#kbd_roving_tabindex) on the `@wordpress/components`' `Toolbar` component and to use it on the header and block toolbars so they become a single tab stop as recommended by the [WAI-ARIA Toolbar Pattern](https://www.w3.org/TR/wai-aria-practices/#toolbar). Related issues are #15331 and #3383.

This PR continues the work done in #20008, but specifically for the **Image**, **Cover** and **Media & Text** blocks.

![GIF showing the Image block while focus is moved using arrow keys and Tab](https://user-images.githubusercontent.com/3068563/83715081-f3e15e00-a602-11ea-82bd-a06bfa4a1cd8.gif)

## How to test

Check Image, Cover and Media & Text blocks toolbars. Use <kbd>Tab</kbd> and arrow keys to move focus.
